### ISSUE TITLE
v2015.5.4 and greater: salt-call --local state.highstate throws "AttributeError: 'module' object has no attribute 'exceptions' "

### ISSUE THREAD

**Issue by tjeason** [saltstack/salt#26827]
For releases v2015.5.4 and v2015.5.5, I am trying to run the command `salt-call --local high.state` on a masterless machine. I am receiving an attribute error stating that the Python module, **exceptions** cannot be used. I am running salt in Ubuntu 12.04 LTS with Python 2.7.3.  Releases under v2015.5.4 works fine. 

Here is my traceback: 

```
[INFO    ] Loading fresh modules for state activity
[INFO    ] Fetching file from saltenv 'base', ** skipped ** latest already in cache 'salt://top.sls'
[INFO    ] Creating module dir '/var/cache/salt/minion/extmods/beacons'
[INFO    ] Syncing beacons for environment 'base'
[INFO    ] Loading cache from salt://_beacons, for base)
[INFO    ] Caching directory '_beacons/' for environment 'base'
[INFO    ] Creating module dir '/var/cache/salt/minion/extmods/modules'
[INFO    ] Syncing modules for environment 'base'
[INFO    ] Loading cache from salt://_modules, for base)
[INFO    ] Caching directory '_modules/' for environment 'base'
[INFO    ] Creating module dir '/var/cache/salt/minion/extmods/states'
[INFO    ] Syncing states for environment 'base'
[INFO    ] Loading cache from salt://_states, for base)
[INFO    ] Caching directory '_states/' for environment 'base'
[INFO    ] Creating module dir '/var/cache/salt/minion/extmods/grains'
[INFO    ] Syncing grains for environment 'base'
[INFO    ] Loading cache from salt://_grains, for base)
[INFO    ] Caching directory '_grains/' for environment 'base'
[INFO    ] Creating module dir '/var/cache/salt/minion/extmods/renderers'
[INFO    ] Syncing renderers for environment 'base'
[INFO    ] Loading cache from salt://_renderers, for base)
[INFO    ] Caching directory '_renderers/' for environment 'base'
[INFO    ] Creating module dir '/var/cache/salt/minion/extmods/returners'
[INFO    ] Syncing returners for environment 'base'
[INFO    ] Loading cache from salt://_returners, for base)
[INFO    ] Caching directory '_returners/' for environment 'base'
[INFO    ] Creating module dir '/var/cache/salt/minion/extmods/outputters'
[INFO    ] Syncing outputters for environment 'base'
[INFO    ] Loading cache from salt://_outputters, for base)
[INFO    ] Caching directory '_outputters/' for environment 'base'
[INFO    ] Creating module dir '/var/cache/salt/minion/extmods/utils'
[INFO    ] Syncing utils for environment 'base'
[INFO    ] Loading cache from salt://_utils, for base)
[INFO    ] Caching directory '_utils/' for environment 'base'
[INFO    ] Loading fresh modules for state activity
[INFO    ] Fetching file from saltenv 'base', ** skipped ** latest already in cache 'salt://common.sls'
[ERROR   ] An un-handled exception was caught by salt's global exception handler:
AttributeError: 'module' object has no attribute 'exceptions'
Traceback (most recent call last):
  File "/usr/bin/salt-call", line 11, in <module>
    salt_call()
  File "/usr/lib/python2.7/dist-packages/salt/scripts.py", line 227, in salt_call
    client.run()
  File "/usr/lib/python2.7/dist-packages/salt/cli/call.py", line 69, in run
    caller.run()
  File "/usr/lib/python2.7/dist-packages/salt/cli/caller.py", line 236, in run
    ret = self.call()
  File "/usr/lib/python2.7/dist-packages/salt/cli/caller.py", line 138, in call
    ret['return'] = func(*args, **kwargs)
  File "/usr/lib/python2.7/dist-packages/salt/modules/state.py", line 538, in highstate
    whitelist=kwargs.get('whitelist')
  File "/usr/lib/python2.7/dist-packages/salt/state.py", line 2977, in call_highstate
    self.serial.dump(high, fp_)
  File "/usr/lib/python2.7/dist-packages/salt/payload.py", line 174, in dump
    fn_.write(self.dumps(msg))
  File "/usr/lib/python2.7/dist-packages/salt/payload.py", line 122, in dumps
    except (OverflowError, msgpack.exceptions.PackValueError):
AttributeError: 'module' object has no attribute 'exceptions'
Traceback (most recent call last):
  File "/usr/bin/salt-call", line 11, in <module>
    salt_call()
  File "/usr/lib/python2.7/dist-packages/salt/scripts.py", line 227, in salt_call
    client.run()
  File "/usr/lib/python2.7/dist-packages/salt/cli/call.py", line 69, in run
    caller.run()
  File "/usr/lib/python2.7/dist-packages/salt/cli/caller.py", line 236, in run
    ret = self.call()
  File "/usr/lib/python2.7/dist-packages/salt/cli/caller.py", line 138, in call
    ret['return'] = func(*args, **kwargs)
  File "/usr/lib/python2.7/dist-packages/salt/modules/state.py", line 538, in highstate
    whitelist=kwargs.get('whitelist')
  File "/usr/lib/python2.7/dist-packages/salt/state.py", line 2977, in call_highstate
    self.serial.dump(high, fp_)
  File "/usr/lib/python2.7/dist-packages/salt/payload.py", line 174, in dump
    fn_.write(self.dumps(msg))
  File "/usr/lib/python2.7/dist-packages/salt/payload.py", line 122, in dumps
    except (OverflowError, msgpack.exceptions.PackValueError):
AttributeError: 'module' object has no attribute 'exceptions'
```

**Comment by jfindlay** [comment]
@tjeason, thanks for the report.  This is a duplicate of #26705.  If you have any further comments, please add them there.

**Comment by tjeason** [comment]
Great. Will there be a newer version (greater than 2015.5.3) that will have fix?

**Comment by jfindlay** [comment]
@tjeason, it will be available in 2015.5.6 and 2015.8.0.
